### ISSUE TITLE
Multiple database servers support

### ISSUE THREAD

**Issue by zhicwu** [zhicwu/docker-biserver-ce#39]
For now the image only support one database server, meaning all three databases including hibernate, jackrabbit and quartz exist in one database server, which is not always true.

**Comment by usbrandon** [comment]
I imagine a possible solution here is to provide some sample docker-compose.yml file using docker secret with changes to the docker-entrypoint.sh script to allow that last mile configuration upon first start up.

https://docs.docker.com/engine/swarm/secrets/#use-secrets-in-compose

I noticed that the conditional expansion operator being used in the script ":=".  Wondering if the intention was to pick up environmental variables and use those, otherwise set defaults.  If so, we might investigate using docker secrets for that.  See what I did in this backup script, the only difference being that secrets are exposed within the docker container at /run/secrets/named_secret

Where I read in the secret that is also used in MariaDB.
https://github.com/usbrandon/docker-pentaho-stack/blob/master/backups/backup_mariadb.sh#L28

**Comment by zhicwu** [comment]
Exactly, but I think it's better to support both secrets and environment variables as the latter one is simpler and easy to run in anywhere.

Yes the := sets default value.
